app/services/articles/enrich_image_attributes.rb

```diff
@@ -39,7 +40,12 @@ def self.call(article)
         img["data-animated"] = true if img_properties.type == :gif
       end
 
-      article.update_columns(processed_html: parsed_html.to_html)
+      if article.main_image && Settings::UserExperience.cover_image_fit == "limit"
+        main_image_size = FastImage.size(article.main_image, timeout: 15)
+        main_image_height = (main_image_size[1].to_f / main_image_size[0]) * 1000 if main_image_size
```

Comment:
Doesn't `FastImage` already return measurement in `[width, height]`?

Reply:
`FastImage` returns height and weight, but the value we want to capture is the value normalized to a height of `1000`. So that's where this calculation comes in.